CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v1.1.0
+------------------------------
+*October 3, 2018*
+
+### Fixed
+- Issue with `!default` causing spacing and breakpoint maps to not be set correctly to not be set correctly (as already set previously, so not overriding properly).
```

Comment:
`to not be set correctly to not be set correctly`

Reply:
To be sure, to be sure 🍀

Reply:
🤦‍♂️